apps/angularlogos/src/app/logos-list/logos-list.component.ts

```diff
@@ -16,6 +16,8 @@ export class LogosListComponent implements OnInit {
   searchTerm$ = new Subject<string>();
   firstSearchTerm$: Observable<string>;
 
+  randomSeed = '?v=' + Math.floor(Math.random() * 10);
```

Comment:
This actually is not a good idea. Because the seed will be different on every page load, which means the browser will never cache the image.

If we could retrieve the date or hash of the last commit and use that as the seed, I think that would be a better idea.

Reply:
To be honest, it doesn't really matter to me whether the logos will be cached or not 😛😉
I'm not sure whether the cache breaks for EVERY request with a query string or only for requests with query string that have not been cached before.
If the latter, a static quary string like `?v=1` would be enough.

Reply:
It's actually not that difficult I think.

If you use this endpoint `https://api.github.com/repos/maartentibau/angular-logos/events`
Just loop through the events and search for the `eventType === 'PushEvent'`

In that result you can use the `id` as your seed.

Reply:
But why this effort anyway? Wouldn't it be enough to use `?v=1` everywhere?
All the old logos will be requested with the exact same URL all the time. The new logos will get the query string as well and this hopefully rebuilds the cache for them

Reply:
I think I'm just misunderstanding this thing, lets give it a try. We can always adapt if need be.

Reply:
You are right, this is definitely a temporary solution, but let's see if it works, then we can explore further in this direction!